driver-core/src/main/com/mongodb/internal/connection/DefaultServerMonitor.java

```diff
@@ -180,8 +179,8 @@ public void run() {
                     }
                     waitForNext();
                 }
-            } catch (MongoInterruptedException e) {
```

Comment:
Why remove this?  Couldn't this exception still be thrown by `lookupServerDescription`?

Reply:
Thanks! It may, it's not obvious which exceptions may be produced by the code in that `try` block.

Reply:
Done.